ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/generators/responseparser/AllocateJobChunkParserGenerator.java

```diff
@@ -35,7 +35,8 @@
     @Override
     public ImmutableList<ResponseCode> toResponseCodeList(
```

Comment:
Alternatively, I could clean up the tests by adding an alternate constructor to the ParseResponse objects, and default to no pagination headers.